hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/tools/ITestMarkerTool.java

```diff
@@ -307,22 +309,25 @@ public void testRunLimitedAudit() throws Throwable {
   }
 
   /**
-   * Run an audit against the landsat bucket.
+   * Run an audit against a bucket with a large number of objects.
    * <p></p>
    * This tests paging/scale against a larger bucket without
    * worrying about setup costs.
    */
   @Test
   public void testRunLimitedLandsatAudit() throws Throwable {
```

Comment:
Should we update the method name? it's still landsat.

Reply:
Good catch, let me update that!